### PR TITLE
More CSS selector logging

### PR DESCRIPTION
**Pull request by stephanfowler**
- send a log after any on-page click event, in addition to on page load
- randomise the delay before sending
- send more in the batch (1000)
- apply to 1 in 5000 users

@rich-nguyen

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/analytics/css-logging.js [hunk 1]
```diff
@@ -64,15 +66,27 @@ define([
...
 
-    return {
-        run: sendReports
+    return function () {
+        var sendAll = window.location.hash === '#csslogging',
+            sender;
+
+        if (sendAll || _.random(1, 5000) === 1) {
+            sender = makeSender(sendAll);
+            sender();
```
Comment (phamann) [inline static/src/javascripts/projects/common/modules/analytics/css-logging.js]: Can't you just write: `sender = makeSener(sendAll)();`

**Comment by phamann** [comment]
:+1: Nice!

**Comment by robertberry** [comment]
:+1: Ship it

File: static/src/javascripts/projects/common/modules/analytics/css-logging.js (hunk 1 above)
Comment (stephanfowler) [inline static/src/javascripts/projects/common/modules/analytics/css-logging.js]: I need `sender` as an unexecuted function, to attach to the event listeners in the following two lines.

**Pull request merged**